File: Java/Herencias/Vaca.java
package Java.Herencias;

public class Vaca extends Mamifero {
    private double litrosLeche;

    public Vaca(String n, double p, double l){
        super(n, p); // Invoca al constructor de la clase padre
        this.litrosLeche = l;
    }

    public void calcularCantidadComida(){
        System.out.println(super.peso*0.03 + "kg de alimento");
    }

    @Override
    public String toString(){
        return super.toString() + "[litros de leche: " + litrosLeche + "]";
    }
}

File: SesionesC#/Herencias/Vaca.cs


public class Vaca : Mamifero 
{
    private double litrosLeche;

    public Vaca(string n, double p, double l) : base(n, p)
    {
        this.litrosLeche = l;
    }

    public void CalcularCantidadComida()
    {
        Console.WriteLine($"{peso * 0.03}kg de alimento");
    }

    public override string ToString()
    {
        return base.ToString() + $"[litros de leche: {litrosLeche}]";
    }
}
